docs/developer/architecture/core/logging-service-configuration.asciidoc

```diff
@@ -0,0 +1,73 @@
+[[logging-service-configuration-example]]
+== Logging Service Configuration Example
+
+As with any configuration in the {kib} platform, logging configuration is validated against the predefined schema and if there are
+any issues with it, {kib} will fail to start with the detailed error message.
+
+Here is a configuration example that can be used to configure _loggers_, _appenders_ and _layouts_:
```

Comment:
I think it'd be nice to have really simple examples for the most basic use cases:
- Log to a file
- Log using ECS format
- Changing the log level

These could be as basic as

```asciidoc
=== Log to a file

Log the default log format to a file instead of to stdout (the default)

[source,yaml]
----
logging:
  appenders:
    file:
      type: file
      fileName: /var/log/kibana.log
      layout:
        type: pattern
root:
  appenders: [file]
----
```

Reply:
++ Love this idea, especially as the new logging config is a bit more verbose, it might be nice to have a couple copy-and-paste recipes folks can use.

I'd propose:
- Log to a file
- Log in json format (basically the same thing as logging with ECS format)
- Include meta in your console logs (since we don't do this by default)
- Change overall log level

Reply:
I've added the basic configuration examples, linked to those from the main `settings` page in `docs/setup` and also added the config example for `elasticsearch` logs.
Hopefully it makes more sense in the flow now.